### PR TITLE
Add Invisible Institute ruleset

### PR DESCRIPTION
**Pull request by VoidPhantom**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
Please add targets:
the.invisible.institute
www.the.invisible.institute

Please note as `Invalid certificate` in a top comment:
cpdb.invisible.institute
data.invisible.institute

File: src/chrome/content/rules/Invisible.institute.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="Invisible Institute">
+	<target host="invisible.institute" />
+	<target host="www.invisible.institute" />
+
+	<rule from="^http:" to="https:" />
+
+	<securecookie host="^invisible\.institute$" name=".+" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Invisible.institute.xml]: `__utma` is under `.invisible.institute`

**Comment by jeremyn** [comment]
@VoidPhantom Are you still interested in working on this?

**Comment by VoidPhantom** [comment]
Sorry for the long absence. :frowning_face:

I added the.invisible.institute. I also found and added securedrop.invisible.institute (this subdomain doesn’t seem to have anything important, but it’s there). www.the.invisible.institute, cpdb.invisible.institute, and data.invisible.institute don’t seem to exist (anymore).

**Comment by jeremyn** [comment]
Please:

- [ ] Change the `ruleset` `name` to `Invisible.institute`

**Comment by jeremyn** [comment]
Thanks, merged.

**Pull request merged**